grammars/json.cson

```diff
@@ -13,10 +13,14 @@
   'schema'
   'topojson'
   'webapp'
+  'config'
```

Comment:
This change needs to go, otherwise we're going to attempt to treat all `*.config` files as JSON, which is definitely not something we want to do.